### PR TITLE
Stars

### PR DESCRIPTION
**Pull request by Qsery93**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,77 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 stars">
+      <span class="stars__star">
+      </span>
```
Comment (valerii-derkach) [inline src/index.html:13]: either
```suggestion
      <span class="stars__star"></span>
```

File: src/index.html [hunk 2]
```diff
@@ -8,6 +8,77 @@
...
+      <span class="stars__star">
+      </span>
```
Comment (valerii-derkach) [inline src/index.html:15]: or
```suggestion
      <span
        class="stars__star"
      >
      </span>
```

**Review by valerii-derkach — CHANGES_REQUESTED** [review]
please provide demolinks
 and fix comments

File: src/index.html (hunk 2 above)
Comment (Qsery93) [reply src/index.html:15]: - [DEMO LINK](https://Qsery93.github.io/layout_stars/)
- [TEST REPORT LINK](https://Qsery93.github.io/layout_stars/report/html_report/)

**Comment by Qsery93** [comment]
- [DEMO LINK](https://Qsery93.github.io/layout_stars/)
- [TEST REPORT LINK](https://Qsery93.github.io/layout_stars/report/html_report/)

>

**Comment by Qsery93** [comment]
> please provide demolinks and fix comments

- [DEMO LINK](https://Qsery93.github.io/layout_stars/)
- [TEST REPORT LINK](https://Qsery93.github.io/layout_stars/report/html_report/)

**Pull request closed without merging**